core/src/main/kotlin/com/exactpro/th2/read/db/bootstrap/Main.kt

```diff
@@ -268,6 +269,7 @@ private fun Any.toStringValue(): String = when (this) {
     is BigDecimal -> stripTrailingZeros().toPlainString()
     is Double -> toBigDecimal().toStringValue()
     is Float -> toBigDecimal().toStringValue()
+    is ByteArray -> toHexString(this)
```

Comment:
Also, I think we should add a test for that. Using `blob` in MySQL should do the thing

Reply:
Added `data` field with `blob` type into DataBaseReaderIntegrationTest